arch/cortex-m/samv71-hal/src/hal.rs

```diff
@@ -0,0 +1,60 @@
+//! System HAL implementation for x86 target.
+
+use aerugo_hal::system_hal::{SystemHal, SystemHardwareConfig};
+use bare_metal::CriticalSection;
+
+use crate::peripherals::Peripherals;
+
+/// HAL implementation for x86.
+pub struct Hal {
+    /// Hardware peripherals.
+    _peripherals: Peripherals,
+}
+
+impl Hal {
+    /// Create new HAL instance.
+    pub const fn new(peripherals: Peripherals) -> Self {
+        Hal {
+            _peripherals: peripherals,
+        }
+    }
+}
+
+impl SystemHal for Hal {
+    type Instant = crate::time::TimerInstantU64<1_000_000>;
+    type Duration = crate::time::TimerDurationU64<1_000_000>;
```

Comment:
move magic number to named constant (if it's related to system clock or configuration, it should later be made tailorable via config or something like that)

Reply:
I'm not 100% sure how does configuring that would look like, so you can treat this as a stub (as this is part of the implementation, not the API). We will do some form of configuration/having this as a common element across the system when we will tackle the TC driver.

Reply:
is it possible to at least move it to a separate constant? It seems like both of these values are supposed to be equal, so it'd make sense to take them from the same source